pandas/io/formats/css.py

```diff
@@ -377,8 +380,20 @@ def _error():
 
     def atomize(self, declarations: Iterable) -> Generator[tuple[str, str], None, None]:
         for prop, value in declarations:
-            prop = prop.lower()
-            value = value.lower()
+            # Need to reparse the value here in case the prop was passed
+            # through as a dict from the styler rather than through this
+            # classes parse method
+            tokens = []
+            for token in tinycss2.parse_component_value_list(value):
+                if token.type == "ident":
+                    # The old css parser normalized all identifier values, do
+                    # so here to keep backwards compatibility
+                    token.value = token.lower_value
```

Comment:
See related discussion here https://github.com/pandas-dev/pandas/pull/48660#discussion_r982922758 regarding case normalization. Seems like its worthy of further investigation of whether this is truly necessary for backwards compatibility

Reply:
I'm not 100%, I looked into this back in the 0.24 code. But I believe the reason was to allow users to set a font like `Arial` and have it work everywhere. The HTML output expects the upper case version, but the Excel library needs an all lower case version.